### PR TITLE
Marie's Bio (hw01) 

### PR DESCRIPTION
**Pull request by mariekfp**
The hardest part of this assignment for me was getting the hang of working with GitHub. To start off, I wasn't entirely certain I understood what it meant to fork something, so I read the description on [GitHub](https://help.github.com/en/articles/fork-a-repo). Once I got that down, I realized I wasn't sure how to make a commit or push my changes to my online GitHub repo. I then searched around the GitHub "help" documentation and read articles like [this one](https://help.github.com/en/articles/about-pull-requests). I lucked out in that a good friend who is in CAPS at Harris happened to also be at the library and could explain GitHub to me. He showed me how many of my issues stemmed from my horrific folder organization. I had to delete and re-fork the assignment a few times, but I eventually got the hang of it (and cleaned up my folders). While I was frustrated I was making so many mistakes, having to repeat the process multiple times definitely helped me better understand what I needed to do / how everything worked. 

Eventually I was able to edit the README.md document, make commits and pushes from the terminal in R, and even push a photo of myself to my GitHub to use in my bio. In the terminal I used the commands git add, git commit -m'comment', and git push. 

At first I was also a bit confused by Markdown. After reading [Chap. 27  in R for Data Science](https://r4ds.had.co.nz/r-markdown.html), I realized I actually have haphazardly used Markdown in the past. I set up a Wikipedia page for a friend last summer. Writing the page required very specific formatting which I now realized is Markdown. Additionally my last company used Slack, and to **bold** or *italicize* text also required Markdown- but I didn't realize that's what it was at the time. I also watched [this](https://rmarkdown.rstudio.com/lesson-1.html) quick tutorial for general overview of Markdown in R Studio. As I was actually writing my bio, I used [this reference guide](https://www.rstudio.com/wp-content/uploads/2015/03/rmarkdown-reference.pdf?fbclid=IwAR0YRVOJezNxinON272gEuz_szvcASWGKiwGbs3YU2VUXa0ki6VmQzQ43F4) in addition to the printed cheatsheet to format my information.

### CODE REVIEW

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
Marie's bio demonstrates a good application of markdown grammar through italics, image, link, and headlines. The content of the bio was clear and informative as well. 
* Some specific praise?
I admire Marie for using git commands in terminal directly instead for Git GUI in Studio. Git commands seem to be much less intuitive and more challenging. 
* Something I learned?
I did not realize that there is a chapter in R for Data Sciences for Markdown. That seems very useful for my own improvement in the future. 
* Specific constructive criticism?
As a new user of Git and R myself, I find it helpful to commit and push more often to ensure my progress was saved. 
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
No really, Marie seems to find solutions to her questions in the assignment.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-# hw01
```
Comment (jfan3) [inline README.md:1]: Nice!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
-Repository for homework 01
+## Marie Pederson
+
+#### MAPSS 2019, Sociology 
+
+![Photo by Matt Libro, Summer 2018](IMG_8229.jpg)
+
```
Comment (jfan3) [inline README.md:6]: Could include sections headers, like `###`

**Review by jfan3 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |x           |              |            |
| **Coding strategy**         |x           |              |            |
| **Presentation**            |           |x              |            |
| **Achievement, creativity** |           |x              |            |
| **Ease of access**          |x           |              |            |

Remarks:

* You HW hit the major features of Markdown.
* I like that there are no extraneous files in the folder.

**Comment by jessrobinson42** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |           |         x     |            |
| **Ease of access**          |       x    |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
Overall, this page was very strong. It seems to meet all the requirements for the homework. It didn't seem to go above and beyond in terms of achievement, but I think that's very hard to do on this sort of assignment.

* Some specific praise?

I really liked how you including which resources you used to learn about Git Hub and R Markdown in your comment. It was really helpful to me, and I imagine other students too. 

* Something I learned?
I learned of some useful tutorials and references. Also, a cool musician!

* Specific constructive criticism?
I think you page was very good, but maybe adding sections, or more links (maybe a link to Rodes Rollins?) might enhance the creativity of your page. 

* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
You seem to have a good grasp on finding solutions to your problems. However, if you're still struggling with using RStudio and Git, I found [this page from the class notes](https://cfss.uchicago.edu/setup/git-with-rstudio/) to be very helpful, though you might have already read it.

**Pull request closed without merging**